marketplace_builder/views/pages/use-cases/building-amp-pages.liquid

```diff
@@ -0,0 +1,69 @@
+---
+converter: markdown
+metadata:
+  title: Building AMP Pages on Platform OS 
+  description: Use case of building AMP pages on Platform OS: standard, benefits, and tutorials to get you started.
+slug: use-cases/building-amp-pages
+searchable: true
+---
+
+This use case describes how you can **build Accelerated Mobile Pages (AMP) on Platform OS**. We explore the benefits of AMP pages, discuss the standard, and link some tutorials to get you started.
+
+## The AMP standard and its benefits 
+Google’s initiative, [AMP](https://www.ampproject.org/learn/overview/) is an open-source library for creating web pages that load almost instantly. Their reduced load time improves user experience, and lowers the bounce rate. Page load time is a factor in search engine optimization, so building AMP pages has a positive effect on search rankings as well. AMP is built to enhance the usability of images and banners, so it can contribute to higher ad viewability rates.   
+
+The AMP standard describes three main aspects:
+
+* **AMP HTML** is HTML extended with custom AMP properties, e.g. some HTML tags are replaced with AMP-specific tags.  
+* **AMP JavaScript** is optimized for performance, for example, it uses external resources asynchronously, sandboxes all iframes, pre-calculates every element on the page before loading resources, and disables slow CSS selectors. 
```

Comment:
I would call them `AMP Components`, to avoid confusion with javascript that can be added by developers - those components: https://www.ampproject.org/docs/reference/components